services/api/src/resources/organization/resolvers.ts

```diff
@@ -358,6 +358,30 @@ export const getProjectsByOrganizationId: ResolverFn = async (
   return rows;
 };
 
+export const getOrganizationByProject: ResolverFn = async (
```

Comment:
I'd just add a comment to this that this resolver should only ever be called by another resolver that has done a permission check on the project first.